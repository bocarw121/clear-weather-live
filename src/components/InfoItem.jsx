import React from "react";
import { StyleSheet, Text, View } from "react-native";

import colors from "../constants/colors";

const InfoItem = ({ description, measure, unit }) => {
  return (
    <View style={styles.container}>
      <Text style={styles.description}>{description}</Text>
      <Text style={styles.unit}>{`${measure} ${unit || ""}`}</Text>
    </View>
  );
};

export default InfoItem;

const styles = StyleSheet.create({
  container: {
    alignItems: "center",
  },
  description: {
    color: colors.text.light,
    fontSize: 20,
    paddingTop: 10,
  },
  unit: {
    color: colors.text.yellow,
    fontSize: 18,
    paddingTop: 10,
  },
});
