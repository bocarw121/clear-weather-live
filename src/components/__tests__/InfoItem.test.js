import React from "react";

import { render } from "@testing-library/react-native";

import InfoItem from "../InfoItem";

it("Should render the InfoItem", () => {
  const screen = render(<InfoItem />);

  expect(screen).toMatchSnapshot();
});
