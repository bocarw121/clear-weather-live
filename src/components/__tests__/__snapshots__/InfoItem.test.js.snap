// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the InfoItem 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 20,
        "paddingTop": 10,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#fdf02a",
        "fontSize": 18,
        "paddingTop": 10,
      }
    }
  >
    undefined 
  </Text>
</View>
`;
